script/process_qrc.py

```diff
@@ -23,31 +23,74 @@
 
 """
 
+# Standard library imports
 from __future__ import absolute_import, print_function
-
+from subprocess import call
 import argparse
 import glob
 import os
 import sys
-from subprocess import call
+
+# Third party imports
+from watchdog.events import FileSystemEventHandler
+from watchdog.observers import Observer
+
+# Constants
+HERE = os.path.abspath(os.path.dirname(__file__))
+REPO_ROOT = os.path.dirname(HERE)
+
+
+class QSSFileHandler(FileSystemEventHandler):
+    """QSS File observer."""
+
+    def __init__(self, parser_args):
+        """QSS File observer."""
+        super(QSSFileHandler, self).__init__()
+        self.args = parser_args
+
+    def on_modified(self, event):
+        """Handle file system events."""
+        if event.src_path.endswith('.qss'):
+            run_process(self.args)
+            print('\n')
 
 
 def main(arguments):
     """Process QRC files."""
     parser = argparse.ArgumentParser(description=__doc__,
                                      formatter_class=argparse.RawDescriptionHelpFormatter)
     parser.add_argument('--qrc_dir',
-                        default='../qdarkstyle',
+                        default=os.path.join(REPO_ROOT, 'qdarkstyle'),
                         type=str,
                         help="QRC file directory, relative to current directory.",)
     parser.add_argument('--create',
                         default='all',
                         choices=['pyqt', 'pyqt5', 'pyside', 'pyside2', 'qtpy', 'pyqtgraph', 'qt', 'qt5', 'all'],
                         type=str,
                         help="Choose which one would be generated.")
+    parser.add_argument('--watch', '-w',
+                        action='store_true',
+                        help="Watch for file changes.")
 
     args = parser.parse_args(arguments)
 
+    if args.watch:
+        path = os.path.join(REPO_ROOT, 'qdarkstyle')
```

Comment:
@dpizetta question 

is `/home/pizetta/workspace/qdark-combo/qdarkstyle/` the repo root or the module?

what does `path` print and what are the contents of that path?

Reply:
I tried something here, started with a fresh clone, and it worked! Running from root, script and qdarkstyle package folder.

Just answering: qdarkstyle is the path to package, the clone directory is qdark-combo (your other branch).

No problems anymore...sorry for that